core/src/main/scala/cell/HandlerPool.scala

```diff
@@ -37,23 +39,23 @@ class HandlerPool(parallelism: Int = 8, unhandledExceptionHandler: Throwable =>
    *
    * @param key The key to resolve this cell if in a cycle or no result computed.
    * @param init A callback to return the initial value for this cell and to set up dependencies.
-   * @param lattice The lattice of which the values of this cell are taken from.
+   * @param updater The lattice of which the values of this cell are taken from.
```

Comment:
"The lattice of which the values of this cell are taken from." -> "The updater used to update the value of this cell."